packages/bpk-component-popover/index.js

```diff
@@ -16,8 +16,13 @@
  * limitations under the License.
  */
 
-import BpkPopoverPortal from './src/BpkPopoverPortal';
+/* @flow */
+
+import BpkPopoverPortal, {
+  type Props as PopoverProps,
+} from './src/BpkPopoverPortal';
 import themeAttributes from './src/themeAttributes';
 
-export default BpkPopoverPortal;
+export type Props = PopoverProps;
```

Comment:
So this is interesting, should we have a policy of doing this to prepare for a future where other people might use Flow?

Reply:
Yeah definitely - if we treat `stories.js` as a consumer then it was a necessity i.e. i couldn't have flow typed that file otherwise.

Reply:
Can we record a decision for that?

Reply:
Yeah sure!!